### PR TITLE
Automated cherry pick of #71522: Use Node-Problem-Detector v0.6.0

### PR DESCRIPTION
**Pull request by wangzhen127**
Cherry pick of #71522 on release-1.10.

#71522: Use Node-Problem-Detector v0.6.0

### CODE REVIEW

File: cluster/gce/gci/configure-helper.sh [hunk 1]
```diff
@@ -1032,6 +1032,11 @@ EOF
 }
 
 function create-node-problem-detector-kubeconfig {
+  local apiserver_address="${1}"
```
Comment (vishh) [inline cluster/gce/gci/configure-helper.sh]: Why is this logic being introduced? Can you add some relevant comments?

**Comment by vishh** [comment]
Kindly resolve the `need code comments` nit prior to submitting.
/approve

File: cluster/gce/gci/configure-helper.sh (hunk 1 above)
Comment (wangzhen127) [reply cluster/gce/gci/configure-helper.sh]: This is to fix https://github.com/kubernetes/node-problem-detector/issues/227. This is consistent with `create-kubelet-kubeconfig`. I think it is the correct and standard way now, so I didn't add any comment.

**Comment by wangzhen127** [comment]
> Kindly resolve the need code comments nit prior to submitting.

@vishh, I replied the comment. Let me know if you still think the comment is needed.

**Comment by wangzhen127** [comment]
Patcher manager, PTAL
/assign @MaciekPytel

**Pull request merged**